content/uc-doc/upgrade/upgrade_notes.md

```diff
@@ -2,6 +2,15 @@
 title: Upgrade notes
 ---
 
+## 21.09 {#21-09}
+
+- It's now mandatory to have the access you want to assign to another resource (i.e. users/groups).
```

Comment:
"You are now required to have the same access you are attempting to assign to another resource (i.e. users/groups)." ?